client/js/transports/transport.mdx

```diff
@@ -3,23 +3,27 @@ title: "Transport Overview"
 sidebarTitle: "Overview"
 ---
 
-Transports are the means by which Pipecat clients communicate with their bot services. They handle both message exchange between client and server and real-time media transport. Pipecat implements the RTVI standard for these communications.
+Transports are the means by which Pipecat clients communicate with their bot services. They handle both message exchange between client and server and real-time media transport.
```

Comment:
There was list of additional functionality (like device management) somewhere else in these docs. Should it go here too?

Reply:
good idea.